pro/src/components/VenueForm/CollectiveVenueInformations/CollectiveDmsTimeline/CollectiveDmsTimeline.tsx

```diff
@@ -233,7 +235,7 @@ const CollectiveDmsTimeline = ({
             waitingAdageStep,
           ]}
```

Comment:
Petite proposition, un ternaire dans un ternaire est plus compliqué à comprendre, dans ce cas ça va, donc à confirmer, peut-être que ce serait plus clair avec quelque chose dans ce style même si c'est pas parfait
```suggestion
          if (!hasAdageId) {
        return (
          <Timeline
            steps={[
              successSubmittedStep,
              successInstructionStep,
              successDoneReferencement,
              waitingAdageStep,
            ]}
          />
        )
      }

      if (!hasAdageIdForMoreThan30Days) {
        return (
          <Timeline
            steps={[
              successSubmittedStep,
              successInstructionStep,
              successDoneReferencement,
              successAdageStep,
            ]}
          />
        )
      }

      return (
        <div className={styles['timeline-added-in-adage']}>
          <ValidCircleIcon />
          <span>Ce lieu est référencé sur ADAGE</span>
        </div>
      )
```